docs/v2/features-overzicht.md

```diff
@@ -163,6 +163,168 @@ In het volgende feature bestand zijn de bovenstaande regels geïllustreerd aan d
 
 - [verblijfstitelvelden vragen met fields](./features/persoon/verblijfstitel/overzicht.feature)
 
+## Vragen om gezag over een minderjarige
+
+Je kan om het gezag over een minderjarige vragen met fields "gezag". Je kan hier niet vragen om alleen specifieke velden binnen gezag, zoals "gezag.minderjarige", je krijgt bij gezag alles of niets.
```

Comment:
demonstreren met feature?

Reply:
@CathyDingemanse mee eens. Ik denk dat er vanuit dit feature overzicht straks een verwijzing moet komen naar:
- features die tonen hoe je gezag moet gebruiken (zoals jij hier denk ik bedoelt)
- document en features die uitleggen hoe gezag bepaald wordt (waar jij nu mee bezig bent)

beide verwijzingen kan ik nu nog niet toevoegen, omdat die features nu nog niet bestaan, dan wel in de Gezag repo staan (niet publiek toegankelijk) in plaats van in de personen repo.